src/unlock/unlocker.rs

```diff
@@ -630,6 +657,11 @@ impl ScriptUnlocker for OmniLockUnlocker {
                 .into(),
             ));
         }
+        if &args.as_ref()[1..21] == H160::default().as_bytes() {
```

Comment:
1. Add some comment describe what is the condition is for
2. Use flag to judge the situation.